src/Compiler/SyntaxTree/ParseHelpers.fs

```diff
@@ -811,3 +811,15 @@ let mkSynMemberDefnGetSet
                 []
         | _ -> []
     | _ -> []
+
+// The last element of elementTypes does not have a star or slash
+let mkTupleOrDivide (isStruct: bool) (elementTypes: TupleTypeSegment list) : SynType =
```

Comment:
I would rename this to `mkSynTypeTuple`, we already moved away from the `divide` thing.

Reply:
Updated. Thanks for  the comment :)